org/urls.py

```diff
@@ -4,15 +4,13 @@
 
 urlpatterns = [
     path('', OrgView.as_view()),
-    path('<int:org_id>/volunteer/',AddVolunteer,name='add_volunteer'),
-    path('<int:org_id>/group/', GroupView.as_view()),
-    path('<int:org_id>/', EditOrg, name='edit_org'),
+    path('<int:org_id>/', EditOrgView.as_view(), name='edit_org'),
     path('<int:org_id>/volunteer/', AddVolunteer, name='add_volunteer'),
+    path('<int:org_id>/group/', GroupView.as_view()),
     path('<int:org_id>/group/', GetGroup, name='get_group'),
```

Comment:
```suggestion
```
line 9 already covers this case

Reply:
I see code duplication in the `upstream/dev` can you check it out?

Reply:
Yup there is duplication. Which will be fixed with this commit. :tada: